akka-docs/src/test/scala/docs/stream/IntegrationDocSpec.scala

```diff
@@ -130,6 +127,74 @@ object IntegrationDocSpec {
     }
   }
   //#ask-actor
+  class ActorRefWithBackPressure {
+    //#actorref-with-ack
+    val initMessage = "start"
+    val onCompleteMessage = "done"
+    val ackMessage = "ack"
+
+    implicit val system = ActorSystem("ActorWithBackPressure")
+    implicit val mat = ActorMaterializer()
+
+    class ActorWithBackPressure(ref: ActorRef) extends Actor {
+      def receive = {
+        case `initMessage` ⇒
+          sender() ! ackMessage
+          ref forward initMessage
+        case `onCompleteMessage` ⇒
+          ref forward onCompleteMessage
+        case msg: Int ⇒
+          sender() ! ackMessage
+          ref forward msg
```

Comment:
It's not very good to use two actors here because then you loose the backpressure and the mailbox of the `MyActor` will fill if that is consuming slower than the stream. Better to just have one actor in this example and send the ack when processing is done.